README.md

```diff
@@ -6,6 +6,7 @@ Telepresence combined with GitHub Actions allows you to run integration tests in
 
 The V0 ediition of github actions for Telepresence contains individual actions to:
 
+1. [Configure](./configure) Telepresence in a new repository. See the action docs to know how to use it.
```

Comment:
"See the action docs to know how to use it.
Write Preview" is a little bit redundant or we could put it after the list. We just want a description of Configure action is.

Reply:
Here we should point to the final documentation page

Reply:
with "here" you mean in this list item of in this README file?